packages/cli/src/commands/update.ts

```diff
@@ -0,0 +1,98 @@
+import axios from "axios";
+import inquirer from "inquirer";
+import semver from "semver";
+import { readFileSync, writeFileSync } from 'fs';
+import { logger } from "../utils/logger.js";
+import { execSync } from "child_process";
+
+export async function updateCommand() {
+    try {
+        const installedPackages = JSON.parse(execSync('npm list --depth=0 --json').toString());
+        const triggerDevPackages = Object.keys(installedPackages.dependencies).filter(pkg => pkg.startsWith('axios'));
+
+        if (triggerDevPackages.length === 0) {
+            logger.info('No @trigger.dev packages found in package.json.');
+            return;
+        }
+
+        const latestVersions = await getLatestVersions(triggerDevPackages);
+        const updates: { [packageName: string]: string } = {};
+
+        triggerDevPackages.forEach(packageName => {
+            const currentVersion = installedPackages.dependencies[packageName].version;
+            const latestVersion = latestVersions[packageName];
+            if (latestVersion) {
+                if (semver.gt(latestVersion, currentVersion)) {
+                    updates[packageName] = latestVersion;
+                }
+            }
+        });
+
+
+        if (Object.keys(updates).length === 0) {
+            console.log('No updates available.');
+            return;
+        }
+
+        const confirm = await confirmUpdate(updates);
+
+        if (confirm) {
+            const updatedPackageJson = updatePackageJson(updates);
+            writeFileSync('package.json', JSON.stringify(updatedPackageJson, null, 2));
+            logger.info('package.json updated.');
+
+            // Run 'npm install' to install the updated packages
+            logger.info('Running npm install...');
+            execSync('npm install', { stdio: 'inherit' });
+            logger.info('Dependencies updated.');
+        } else {
+            logger.info('Update canceled.');
+        }
+    } catch (error) {
+        logger.error('An error occurred:', error);
+    }
+}
+
+async function getLatestVersions(packageNames: string[]): Promise<{ [packageName: string]: string }> {
```

Comment:
Some of our packages aren't currently using the "latest" tag (like `@trigger.dev/sdk`, `@trigger.dev/github`, `@trigger.dev/slack` and `@trigger.dev/resend`) but instead are using the `next` tag. Could this code be updated to update packages that are tagged with `next` to either:

1. The more latest `next` version, if it exists
2. The more latest `latest` version, if it exists and is considered "later" in semver standards

Reply:
Can you review the current logic to see that i understood correctly?